### ISSUE TITLE
Running on macOS?

### ISSUE THREAD

**Issue by MichaelVoelkel** [egor-tensin/build-boost#3]
It does not seem to work on macOS or am I doing something wrong here?

I'm getting

```
  shell: /usr/local/bin/pwsh -command ". '{0}'"
110
  env:
111
    BUILD_TYPE: Release
113
boost-download: /Users/runner/work/_temp/c0677176-fc7f-4b75-9376-a283096882c9.ps1:38
114
Line |
115
  38 |  boost-download --cache $base_dir -- $version $boost_dir
116
     |  ~~~~~~~~~~~~~~
117
     | The term 'boost-download' is not recognized as a name of a
118
     | cmdlet, function, script file, or executable program. Check
119
     | the spelling of the name, or if a path was included, verify
120
     | that the path is correct and try again.
```

**Comment by egor-tensin** [comment]
@MichaelVoelkel Hello, thanks for bringing this up. I don't have an Apple device, which is why I was hesitant to test on macOS. I'll check it out.

**Comment by MichaelVoelkel** [comment]
Great, thank you!

I forked your repository and tried my luck by running your tests. It seems to fail because `boost-download` from `cmake-common` cannot really be called. Python is installed, permissions to execute are there, but for some reason it still does not work... I tried to replace direct call with `StartProcess`, but my powershell knowledge seems too limited at the moment, so I gave up after a while because it did not work due to other errors.

But would we great if you found a way to get it working.

**Comment by egor-tensin** [comment]
Sorry for the delay, I'm completely swamped with work. It's fixed now in version v1.2 (v1 points to it now). The earliest version of Boost to build successfully was 1.71.0 though. Please report back if you see any issues.

Proof: the macOS jobs at the bottom of https://github.com/egor-tensin/build-boost/actions/runs/1582771726.
